businessRules/ideas.js

```diff
@@ -1,91 +1,76 @@
-const { getAllIdeas, createIdea, updateIdea } = require('../models/ideas');
+const { getIdeas, createIdea, updateIdea } = require('../models/ideas');
+const {
+    validateHeader,
+    validateDescription,
+    validateAskedSum,
+    validateIdea,
+} = require('../validations/ideas');
+const { validateUserId } = require('../validations/users');
 
 const IDEA_STATUS = {
     pending: 'pending',
     accepted: 'accepted',
     rejected: 'rejected',
 };
 
-function getIdeas() {
-    return getAllIdeas();
-}
-
-function sortedByDonationSumIdeas() {
-    const ideasList = getIdeas();
-
-    const sortedIdeasList = ideasList.sort(
+function sortIdeasByTotalDonationSum(ideas) {
+    const sortedIdeasList = ideas.sort(
```

Comment:
The word `List` in `sortedIdeasList` is redundant

Reply:
Fixed